app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -84,6 +84,16 @@ def post_analytics(result)
       analytics.multi_factor_auth_setup(**properties) if context == 'confirmation'
 
       analytics.track_mfa_submit_event(properties)
+
+      if UserSessionContext.authentication_context?(context) || UserSessionContext.reauthentication_context?(context)
+        irs_attempts_api_tracker.mfa_phone_verification_otp_submitted(
+          success: properties[:success],
```

Comment:
Should we use the same `reauthentication` parameter here? https://github.com/18F/identity-idp/pull/6692

Reply:
implemented :)